### PR TITLE
zephyr: tester: MCP tests

### PR DESCRIPTION
**Pull request by piotrnarajowski**
Adding missing handler for MCP/SR/SP/BV-01-C testcase. Although this is MCP testcase, it uses Generic Media Control Service - merge after [PR#1054](https://github.com/auto-pts/auto-pts/pull/1054)

### CODE REVIEW

**Comment by piotrnarajowski** [comment]
@mkasenberg

File: doc/btp_gmcs.txt [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+		than required bytes received) it shall be assumed that command
+		is not supported.
+
+		In case of an error, the error response will be returned.
+
+	Opcode 0x02 - Command Send
+
+		Controller Index:	<controller id>
+		Command parameters:	Opcode (1 octets)
+					Use Param (1 octet)
```
Comment (mkasenberg) [inline doc/btp_gmcs.txt:26]: `Use Param` is boolean or data length of `Param`? Please add a little annotation here.

**Review by mkasenberg — CHANGES_REQUESTED** [review]

File: doc/btp_gmcs.txt (hunk 1 above)
Comment (piotrnarajowski) [reply doc/btp_gmcs.txt:26]: Use param is a boolean

File: autopts/ptsprojects/zephyr/mcp.py [hunk 2]
```diff
@@ -89,10 +89,21 @@ def test_cases(ptses):
...
+            instance = ZTestCase("MCP", tc_name,
+                                 cmds=pre_conditions_server,
+                                 generic_wid_hdl=mcp_wid_hdl)
+            tc_list.append(instance)
+            continue
+        else:
+            instance = ZTestCase("MCP", tc_name,
+                                 cmds=pre_conditions,
+                                 generic_wid_hdl=mcp_wid_hdl)
+            tc_list.append(instance)
```
[the remark this answers is not in the document]
Comment (piotrnarajowski) [reply autopts/ptsprojects/zephyr/mcp.py]: Fixed

**Review by mkasenberg — APPROVED** [review]

**Pull request merged**